src/main/java/no/nav/veilarbperson/service/PdlService.java

```diff
@@ -1,29 +1,188 @@
 package no.nav.veilarbperson.service;
 
-import no.nav.veilarbperson.client.pdl.HentPersonData;
+import lombok.extern.slf4j.Slf4j;
+import no.nav.common.client.norg2.Norg2Client;
+import no.nav.common.types.identer.Fnr;
+import no.nav.veilarbperson.client.dkif.DkifClient;
+import no.nav.veilarbperson.client.dkif.DkifKontaktinfo;
+import no.nav.veilarbperson.client.egenansatt.EgenAnsattClient;
+import no.nav.veilarbperson.client.pam.PamClient;
+import no.nav.veilarbperson.client.pdl.HentPdlPerson;
 import no.nav.veilarbperson.client.pdl.PdlClient;
+import no.nav.veilarbperson.client.pdl.PdlPersonData;
+import no.nav.veilarbperson.client.pdl.domain.Familiemedlem;
+import no.nav.veilarbperson.client.person.PersonClient;
+import no.nav.veilarbperson.client.veilarbportefolje.Personinfo;
+import no.nav.veilarbperson.client.veilarbportefolje.VeilarbportefoljeClient;
+import no.nav.veilarbperson.domain.Enhet;
+import no.nav.veilarbperson.domain.PersonData;
+import no.nav.veilarbperson.utils.PdlPersonDataMappper;
+import no.nav.veilarbperson.utils.PersonDataMapper;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Service;
 
+import java.util.Collection;
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Collectors;
+
+import static no.nav.veilarbperson.utils.Mappers.fraNorg2Enhet;
+
+@Slf4j
 @Service
 public class PdlService {
```

Comment:
Ville nok fjernet PDLService også og heller flyttet logikken til PersonService. Trenger kun litt ekstra kode for å hente data fra PDL og flette inn info som før